src/test/java/org/minions/devfund/mauricioz/linkedList/ContainsTest.java

```diff
@@ -0,0 +1,344 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+import java.util.LinkedList;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ Source code recreated from a .class file by IntelliJ IDEA.
+ (powered by Fernflower decompiler)
+ */
+public class ContainsTest {
+    private static final int LIST1FIRSTVALUE = 4;
+    private static final int LIST1SECONDVALUE = 8;
+    private static final int LIST1THIRDVALUE = 3;
+    private static final int LIST1FOURTHVALUE = 12;
+    private static final int LIST1FIFTHVALUE = 6;
+    private static final int LIST2EXTRAVALUE1 = 5;
+    private static final int LIST2EXTRAVALUE2 = 7;
+    private static final int LIST2EXTRAVALUE3 = 9;
+    private static final int LIST2EXTRAVALUE4 = 2;
+    private static final String MSG1 = "containsSubsequence returns true when first LinkedList contains all elements";
+    private static final String MSG2 = "when first LinkedList contains second";
+    private static final String MSG3 = "containsSubsequence throws";
+
+    /**
+     * test.
+     */
+    @Test
+    public void testContainsMiddle() {
+        LinkedList var1 = new LinkedList();
+        LinkedList var2 = new LinkedList();
+        var1.add(LIST1FIRSTVALUE);
+        var1.addLast(LIST1SECONDVALUE);
+        var1.addLast(LIST1THIRDVALUE);
+        var1.addLast(LIST1FOURTHVALUE);
+        var1.addLast(LIST1FIFTHVALUE);
+        var2.add(LIST1FIRSTVALUE);
+        var2.add(LIST1SECONDVALUE);
+        var2.add(LIST1THIRDVALUE);
+
+        try {
+            boolean var3 = LinkedListUtils.containsSubsequence(var1, var2);
+            Assert.assertTrue("containsSubsequence returns false " + MSG2 + " in "
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "containsSubsequence returns false " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)